### PR TITLE
lesson_4_1

### PR DESCRIPTION
**Pull request by akhhanna20**
Changed some styles, added button Edit and Drag&Drop

### CODE REVIEW

File: src/components/TodoList.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  //Type-checking tool for props
+  TodoList.propTypes = {
+    todoList: PropTypes.array,
+    onRemoveTodo: PropTypes.func,
+    handleCheckboxChange: PropTypes.func,
+    handleDrag: PropTypes.func,
+    handleDrop: PropTypes.func,
+    id: PropTypes.string,
+    onUpdateNewTitle: PropTypes.func,
+  };
```
Comment (gsvidal) [inline src/components/TodoList.js]: It'd be a better practice to move the PropTypes outside the TodoList component (and the other components as well), doing this you ensure that it's declared once and not each time TodoList function is called. Also adding .isRequired to each property in propTypes is a good practice if you want to ensure that those props are always provided. It helps catch errors early during development and communicates the required nature of these props to other developers who might use your component.

**Review by gsvidal — CHANGES_REQUESTED** [review]
Good work, I left you a couple of suggestions to improve the code though.

File: src/components/TodoList.js (hunk 1 above)
Comment (akhhanna20) [reply src/components/TodoList.js]: Done

**Review by gsvidal — APPROVED** [review]
Well done, everything works as expected.

**Pull request merged**